ports/esp32/main/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
+if(NOT MICROPY_PORT_DIR)
+   get_filename_component(MICROPY_PORT_DIR ${CMAKE_CURRENT_LIST_DIR}/.. ABSOLUTE)
+endif()
+
```

Comment:
Do you think it makes sense to put this down below, after setting `MICROPY_DIR`, and then defaulting `MICROPY_PORT_DIR` to `${MICROPY_DIR}/ports/esp32`?  That way, someone can change MICROPY_DIR and the port dir will follow.

Reply:
That works for me.